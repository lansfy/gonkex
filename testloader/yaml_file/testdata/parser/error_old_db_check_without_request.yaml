- name: WHEN use dbResponse without dbQuery MUST fail with error
  method: GET
  path: /some/path
  response:
    200: "aaabbb"
  dbResponse:
    - item1
