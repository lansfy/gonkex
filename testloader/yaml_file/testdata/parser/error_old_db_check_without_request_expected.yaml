- Error: "process 'testdata/parser/error_old_db_check_without_request.yaml': test 'WHEN use dbResponse without dbQuery MUST fail with error': 'dbResponse' found without corresponding 'dbQuery'"
